bluemix/configuration/core_config/bx_config.go

```diff
@@ -154,7 +154,10 @@ func (c *bxConfig) writeRaw(cb func()) {
 
 	err := c.persistor.Save(c.data.raw)
 	if err != nil {
-		c.onError(err)
+		c.onError(err) // NOTE: the error could be triggered by a file-locking issue,
+		// a file-unlocking issue, OR a file-writing issue; currently, the error chain
+		// ends in `panic("configuration error: " + err.Error()"`, which is somewhat
+		// generic but sufficient for all three of these error types
```

Comment:
Nit: For readability can we move the comment above line 157.

Reply:
Sure!